features/word-break-break-word.yml

```
@@ -0,0 +1,11 @@
+name: "`word-break: break-word`"
+description: "The `word-break: break-word` CSS declaration sets word breaks to occur according to their customary rules. It is superceded by `overflow-wrap: anywhere`"
```

I think this is more inline with our "not to be confused with" and "also known as". I suspect we'll soon need a style guide entry for these.

```suggestion
description: "The `word-break: break-word` CSS declaration sets word breaks to occur according to their customary rules. Superseded by `overflow-wrap: anywhere`"
```